homeassistant/components/homekit_controller/alarm_control_panel.py

```diff
@@ -28,13 +28,19 @@
 }
 
 
-def setup_platform(hass, config, add_entities, discovery_info=None):
```

Comment:
I would keep an `async_setup_platform` that just `pass` around, just to avoid a stack trace if a user would set up a platform from a config section by mistake.
```py
async def async_setup_platform(
        hass, config, async_add_entities, discovery_info=None)
    """Legacy set up platform."""
    pass
```